templates/cohorts/cohort_details.html

```diff
@@ -241,20 +241,20 @@ <h4 class="panel-title">Selected Filters</h4>
                 <h4 class="panel-title">Details</h4>
             </div>
             <div class="panel-body">
-                <div class="single-line">
+                <div class="row col-md-12 row-spacer-v">
```

Comment:
Doesn't this make each div full width now? 

Also I have some helper classes for spacing in the basic.sass file (lines 144-170).